spras/config.py

```diff
@@ -222,16 +223,36 @@ def process_config(self, raw_config):
                 if cur_params[run_params]:
                     for p in cur_params[run_params]:
                         param_name_list.append(p)
-                        all_runs.append(eval(str(cur_params[run_params][p])))
+                        obj = str(cur_params[run_params][p])
+                        try:
+                            obj = [int(obj)]
+                        except ValueError:
+                            try:
+                                obj = [float(obj)]
+                            except ValueError:
+                                if obj.startswith(("range", "np.linspace", "np.arange", "np.logspace", "[")):
+                                    obj = eval(obj)
+                                else:
+                                    obj = [obj]
+                            if not isinstance(obj, Iterable):
+                                raise ValueError(f"The object `{obj}` in algorithm {alg['name']} at key '{p}' in run '{run_params}' is not iterable!") from None
```

Comment:
This behavior seems inconsistent with the numeric parameter behavior that is now supported. We can have a int or float that is not wrapped in a list, but a string still has to be wrapped in a list. Users won't understand that.

Should the case here treat the parameter as a single string? Are there downsides I'm missing?